doc/Developing.md

```diff
@@ -338,6 +338,8 @@ Using Git Bash as administrator:
 
 If you do not have Git Bash installed on your system, [get one](https://gitforwindows.org/), or use `scoop`: `scoop install git`.
 
+If you want to run the electron based example then make sure to re-build the native packages by running `yarn run rebuild:electron` from the root directory. Note that if you wish to switch back to the browser example then you have to roll back these changes by running `yarn run rebuild:browser`.
```

Comment:
I don't think we need to repeat ourself. There is TOC and there is a section explaining how to run Electron which says one must run `yarn run rebuild:electron` before. If someone skips reading docs and goes to troubleshooting it is strange. Maybe just add a reminder instead that read docs first. The same in linux section.

Reply:
Ok @akosyakov . So, shall I replace this with the following line

```
For running the electron example follow the (Running electron-based example)[https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#run-the-browser-based-example-application] section above.
```
or shall I completely remove it?

Also, should I keep the error message in the troubleshooting example below? and just add `First make sure that you follow the steps given in the docs correctly.`

Reply:
I would prefer to remove it completely and add a notice at the start of `Troubleshooting`

> First make sure that you follow the steps given in the docs correctly.

You can use `>` before to highlight it.

Reply:
Done!